_modules/dotnet-cli.js

```diff
@@ -25,15 +25,20 @@ const { red, tabbedNewLine } = formatters;
 // -----------------------------------------------------------------------------------------
 
 const dotnetCli = {
-    cmd() {
-        if (!dotnetPath.cliPath()) return "";
-        return `dotnet ${path.basename(dotnetPath.cliPath())}`;
+    cmd(cliArgs) {
+        return {
+            args: [path.basename(dotnetPath.cliPath() || "")].concat(cliArgs.split(" ")),
```

Comment:
Suggestion: Consider moving `path.basename(dotnetPath.cliPath() || "")` into a local variable above the return. Would make it easier to digest